### ISSUE TITLE
Empty initial restart file

### ISSUE THREAD

**Issue by kieranricardo** [coecms/access-esm-amip#1]
The restart file used by the config (`/g/data/access/payu/access-esm/restart/amip/restart_dump.astart`) is empty. The other restart file `restart_dump.astart.orig` has too many prognostic fields but after running 

```
module load python2
python ~access/apps/pythonlib/umfile_utils/um_fields_subset.py -x 95,171,172,173,174,176,177,178,179,180,181,184,185,186,187,188,189,192,250,413,414,415,416,33001,33002 -I restart_dump.astart.orig -o restart_dump.astart
```

the model now runs correctly. 

I can copy over my new restart file to `/g/data/access/payu/access-esm/restart/amip` but I just wanted to check before I started moving/deleting things!

**Comment by kieranricardo** [comment]
@HoWol76

**Comment by paolap** [comment]
Holger is on leave, so he won't be able to answer until next week

**Comment by kieranricardo** [comment]
No worries, is there anyone else I can ask in the meantime? Holger was just the only person I new to ask!

**Comment by paolap** [comment]
Not really, you could try posting the question in the access-nri slack, but they don't really do support

**Comment by kieranricardo** [comment]
I'm actually from access-nri!

no problem I'll just wait for Holger to get back - cheers :)

**Comment by paolap** [comment]
sorry I would have suggested Aidan but he's on leave, unfortunately all the people who might have some knowledge either are on leave or have moved to other jobs :-)
